tests/suite-unit/PluginFlyvemdmTaskstatus.php

```diff
@@ -35,6 +35,40 @@
 
 class PluginFlyvemdmTaskstatus extends CommonTestCase {
 
+    private $dataField = [
+        'group'     => 'application',
+        'symbol'    => 'deployApp',
+        'type_data' => '',
+        'unicity'   => 0,
+     ];
+     /**
+      * @return array
+      */
+     private function createNewTaskInstance() {
+        $class = new \PluginFlyvemdmTask();
+        $task = $this->newTestedInstance($class);
+        return $task;
+     }
+
+     public function beforeTestMethod($method) {
+        switch ($method) {
+            case 'testPrepareInputForUpdate':
+                parent::beforeTestMethod($method);
```

Comment:
I would prefer to move this before the switch (){...}.

Also, if I remember correctly, no need to call yourself ```this->setupGLPIFramework();```